modyn/selector/selector.py

```diff
@@ -111,14 +112,11 @@ def uses_weights(self) -> bool:
         return self._strategy.uses_weights
 
     def get_selection_strategy_remote(self) -> tuple[bool, str, dict]:
-        assert not (
-            self._strategy._requires_remote_computation and not isinstance(self._strategy, AbstractDownsampleStrategy)
-        )
-
-        if self._strategy._requires_remote_computation and isinstance(self._strategy, AbstractDownsampleStrategy):
+        if isinstance(self._strategy, CoresetStrategy):
+            self._strategy.inform_next_trigger()
```

Comment:
similarily, here. asking the selector what strategy to use should not inform somebody about something. this is a getter and not an "hey selector something happened" function. We have a on_trigger function that gets called per trigger. Why don't we use that?